test/unit/utils/password-validation.test.ts

```diff
@@ -0,0 +1,16 @@
+import { expect } from "chai";
+import { describe } from "mocha";
+import {
+    isCommonPassword,
+} from "../../../src/utils/password-validation";
+
+describe("password-validation", () => {
+    describe("commonPassword", () => {
+    it("should return true when password is in common password list", () => {
+        expect(isCommonPassword("password")).to.equal(true);
+      });
+    it("should return false when password is not in common password list", () => {
+    expect(isCommonPassword("arthnfk174_g2")).to.equal(false);
```

Comment:
what happens if the password is null or ""?

Reply:
It wouldn't get that far due to the ordering of the `create-password-validation` we check for lengths etc before checking for common passwords